server/routes/bookVisit/checkVisitDetailsController.ts

```diff
@@ -43,7 +43,12 @@ export default class CheckVisitDetailsController {
         delete req.session.bookingJourney
         return res.redirect(paths.BOOK_VISIT.BOOKED)
       } catch (error) {
-        // TODO handle errors (VB-3597)
+        // HTTP 400 Bad Request is the response when a session is no longer available
+        if (error.status === 400) {
+          req.flash('message', 'Your visit time is no longer available. Select a new time.')
```

Comment:
This string is duplicated in a few places but this will be handled better when text moved into the il8n framework (coming soon...)